backend/xivdpscalc/types.py

```diff
@@ -0,0 +1,4 @@
+""" Types and type aliases to be used across xivdpscalc """
+
+ElapsedTime = float
+CastTime = float
```

Comment:
Sorry about such a late review item, but pylint is complaining that there's no newline at the end here.